### ISSUE TITLE
Undefined references are not properly displayed

### ISSUE THREAD

**Issue by nilshg** [JunoLab/atom-julia-client#44]
In the REPL:
````
julia> Array(AbstractString,1)
1-element Array{AbstractString,1}:
 #undef
````

In Juno:
````
UndefRefError: access to undefined reference
 in yieldto at task.jl:71
 in wait at task.jl:371
 in wait at task.jl:286
 in wait at task.jl:108
 [inlined code] from C:\Users\tew207\.julia\v0.4\Lazy\src\dynamic.jl:69
 in anonymous at C:\Users\tew207\.julia\v0.4\Atom\src\eval.jl:50
 in handlemsg at C:\Users\tew207\.julia\v0.4\Atom\src\comm.jl:63
 [inlined code] from C:\Users\tew207\.julia\v0.4\Atom\src\comm.jl:18
 in anonymous at task.jl:63
````
This problem extends to any Array containing an `#undef` element.

**Comment by Varanas** [comment]
This is defintely annoying and I don't know how to solve it without calling `isdefined` on each displayed element. Might be worth having a look ath the REPL code in `base`.

**Comment by one-more-minute** [comment]
I think calling `isdefined` basically is the solution – that's how it was fixed in LT, anyway.

**Comment by one-more-minute** [comment]
Fixed by #18
